home/pi/bin/mycroft-wipe

```diff
@@ -68,6 +68,10 @@ case $key in
       echo "update_config=1" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf > /dev/null
       echo "country=US" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf > /dev/null
 
+      # wipe ssh keys and regenerate new ones
+      sudo rm /etc/ssh/ssh_host_*
+      sudo dpkg-reconfigure openssh-server
```

Comment:
Let's do this dpkg-reconfigure on the next boot as part of the startup script, not at the time of the wipe.  Then we can zap keys and shutdown with this and capture a fresh Picroft image.  When a user pulls it down and initially boots, the console session will recreate the missing keys.